### PR TITLE
pango: add a workaround for the broken GlyphVisAttr

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
cc @sdroege @GuillaumeGomez

### CODE REVIEW

**Comment by bilelmoussaoui** [comment]
See here https://github.com/gtk-rs/gtk-rs-core/pull/306/commits/134cee888a4980b9249a050822a701fe0ba69bbe for the end-result post this change.

File: fix.sh [hunk 1]
```diff
@@ -111,6 +111,12 @@ xmlstarlet ed -L \
 	-u '//_:type[@c:type="hb_feature_t*"]/@c:type' -v "gconstpointer" \
 	Pango-1.0.gir
 
+# Fix broken bitfield on pango
```
Comment (sdroege) [inline fix.sh:114]: It's not broken, just not supported by gir :) Also I suggested to replace the content of `PangoGlyphVisAttr` accordingly. That should be sufficient.

Comment (bilelmoussaoui) [reply fix.sh]: Right, I rephrased the comment. Although, I'm not sure why you want to replace the whole content of `PangoGlyphVisAttr` ?

Comment (bilelmoussaoui) [reply fix.sh]: It's only used in GlyphInfo btw

Comment (sdroege) [reply fix.sh]: Yes but like that you would keep the structure of the types at least

Comment (bilelmoussaoui) [reply fix.sh]: Right, but I'm unsure how to handle it then. What I'm supposed to modify in the VisAttr?

Comment (bilelmoussaoui) [reply fix.sh]: never mind, should be fixed now

File: Pango-1.0.gir [hunk 2]
```diff
@@ -6171,15 +6171,15 @@ the shaping and rendering phases.
...
+      <field name="is_cluster_start" writable="1">
         <doc xml:space="preserve">set for the first logical glyph in each cluster.
   (Clusters are stored in visual order, within the cluster, glyphs
   are always ordered in logical order, since visual order is meaningless;
   that is, in Arabic text, accent glyphs follow the glyphs for the
   base character.)</doc>
         <type name="guint" c:type="guint"/>
       </field>
-      <field name="is_color" writable="1" bits="1">
+      <field name="is_color" writable="1">
```
Comment (sdroege) [inline Pango-1.0.gir:6182]: One of the two fields has to go away, otherwise the struct has twice the size it should

Comment (bilelmoussaoui) [reply Pango-1.0.gir]: I'm not sure whether it matters at the end (as nothing uses it) other than maybe -sys tests. But should be fixed now

**Review by sdroege — APPROVED** [review]

**Pull request merged**